src/main/java/mate/academy/springboot/datajpa/dto/mapper/DtoMapper.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.springboot.datajpa.dto.mapper;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public interface DtoMapper<T, T1, T2> {
```

Comment:
if you want to use an interface for the mapper, it's better to split it into two separate interfaces - for request and response because a mapper with 3 parameters looks strange. You can leave one implementation that implements two interfaces